contrib/release/README.md

```diff
@@ -39,4 +39,52 @@ a PR to specify the release note text.
    `relnotes` with the `--verbose` flag to see individual decision taken for
    each PR.
 
+## uploadrev
+
+The `uploadrev` script takes a git revision as the only argument and uploads it
+to releases.cilium.io. For the script to work AWS credentials are required.
+Please see the [AWS CLI documentation][2] for configuration. 
```

Comment:
Please add a paragraph here what this script actually uploads. It is not obvious that it uploads all binaries.

Reply:
Since this is already applied, I will do that as a follow-up. Thanks.